apps/web/components/result/Result.tsx

```diff
@@ -1,25 +1,26 @@
 // 3rd party
 import { Heading } from "react-aria-components";
 // repo
-import useWalkthroughData, {
-  RelatedWalkthroughsType,
-} from "@repo/data/useWalkthroughData";
+// import useWalkthroughData, {
```

Comment:
same question about these comments, making sure everything is explicitly commented out for a reason, easy to accidentally comment something out that could just be deleted

Reply:
Yeah, same as above. More a note in the file and the commented out imports are just because they're not used currently. I figured I would clean up when I revisited.